editor/src/messages/portfolio/document/document_message_handler.rs

```diff
@@ -143,6 +143,29 @@ impl MessageHandler<DocumentMessage, (u64, &InputPreprocessorMessageHandler, &Pe
 								);
 							}
 							DocumentResponse::DocumentChanged => responses.push_back(RenderDocument.into()),
+							DocumentResponse::DeletedSelectedManipulatorPoints => {
+								// Clear Properties panel after deleting all points
+								responses.push_back(
+									FrontendMessage::UpdatePropertyPanelOptionsLayout {
+										layout_target: LayoutTarget::PropertiesOptions,
+										diff: vec![WidgetDiff {
+											widget_path: vec![],
+											new_value: DiffUpdate::SubLayout(vec![]),
+										}],
+									}
+									.into(),
+								);
+								responses.push_back(
+									FrontendMessage::UpdatePropertyPanelSectionsLayout {
+										layout_target: LayoutTarget::PropertiesSections,
+										diff: vec![WidgetDiff {
+											widget_path: vec![],
+											new_value: DiffUpdate::SubLayout(vec![]),
+										}],
+									}
+									.into(),
+								);
+							}
```

Comment:
This doesn't update the backend's widget state, which may cause bugs with the widget diffing. The standard way to clear a panel is with the `SendLayout` message, although this isn't currently very obvious due to lack of documentation (which we are aiming to improve).
```rs
responses.push_back(
	LayoutMessage::SendLayout {
		layout: Layout::WidgetLayout(WidgetLayout::new(vec![])),
		layout_target: LayoutTarget::[your layout target],
	}
	.into(),
);
```